gui/cefdisplay/CMakeLists.txt

```diff
@@ -21,59 +21,34 @@ file(MAKE_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 
 if(ROOT_ARCHITECTURE MATCHES macosx)
    set(CEF_platform src/gui_handler_mac.mm)
-   set(CEF_RESOURCES $ENV{CEF_ROOT}/Release/Chromium\ Embedded\ Framework.framework/Resources)
-   set(CEF_BLOBS ${CEF_RESOURCES})
-   set(CEF_LIBRARY ${CEF_root}/Release/Chromium\ Embedded\ Framework.framework/Chromium\ Embedded\ Framework)
+   set(CEF_RELEASE_DIR ${CEF_root}/Release/Chromium\ Embedded\ Framework.framework)
+   set(CEF_LIBRARY ${CEF_RELEASE_DIR}/Chromium\ Embedded\ Framework)
    set(CEF_LIB_DEPENDENCY "-framework Foundation")
    set(CEF_DLL_WRAPPER ${CEF_root}/build/libcef_dll_wrapper/libcef_dll_wrapper.a)
+   file(MAKE_DIRECTORY ${CMAKE_BINARY_DIR}/Frameworks/Chromium\ Embedded\ Framework.framework)
+   file(COPY ${CEF_RELEASE_DIR} DESTINATION ${CMAKE_BINARY_DIR}/Frameworks/)
 elseif(MSVC)
    set(CEF_platform src/gui_handler_win.cc)
    set(CEF_RESOURCES ${CEF_root}/Resources)
    set(CEF_RELEASE_DIR ${CEF_root}/Release)
-   set(CEF_BLOBS ${CEF_RELEASE_DIR})
    set(CEF_LIBRARY ${CEF_RELEASE_DIR}/libcef.lib)
    set(CEF_DLL_WRAPPER ${CEF_root}/build/libcef_dll_wrapper/Release/libcef_dll_wrapper.lib)
-   configure_file(${CEF_RELEASE_DIR}/libcef.dll ${CMAKE_BINARY_DIR}/bin/ COPYONLY)
+   file(COPY ${CEF_RELEASE_DIR}/ DESTINATION ${CMAKE_LIBRARY_OUTPUT_DIRECTORY})
+   file(COPY ${CEF_RESOURCES}/ DESTINATION ${CMAKE_LIBRARY_OUTPUT_DIRECTORY})
```

Comment:
So the DLL will be copied in the lib directory? Or did I miss something?

Reply:
I made same approach as on Linux and Mac, but one should test it on real Windows.
Unfortunately, my Windows VM is not working - there is problem with latest Linux Kernel
Once I resolve it, I will try to build on Windows as well

Reply:
So if the content of `${CEF_RELEASE_DIR}` are DLLs, please copy them in `${CMAKE_RUNTIME_OUTPUT_DIRECTORY}` (which should be `${ROOTSYS}/bin` instead of `${CMAKE_LIBRARY_OUTPUT_DIRECTORY}`

Reply:
`${CEF_RESOURCES}` contains different translation/coding packages for chrome. 
In 2018 all these files need to be copied to binary directories (on Linux, Windows and Mac), but now behavior was changed on Linux and Mac. I will adjust PR soon

Reply:
I modify PR as you requested